pamai/agents/denet.py

```diff
@@ -255,13 +255,13 @@ def visualize(self):
             }
 
             # Visualize the data
-            vis = sed_vis.visualization.EventListVisualizer(event_lists=event_lists,
-                                                        audio_signal=audio_container.data,
-                                                        sampling_rate=audio_container.fs)
+            # vis = sed_vis.visualization.EventListVisualizer(event_lists=event_lists,
+            #                                             audio_signal=audio_container.data,
+            #                                             sampling_rate=audio_container.fs)
```

Comment:
Is this not working? Could you add a simple comment on why we are not using this anymore? 🤗